README.md

```diff
@@ -142,6 +142,19 @@ export WEB3_PROVIDER_URI=<YOUR WEB3 PROVIDER URI>
 export NUCYPHER_NETWORK=<NETWORK NAME>
 ```
 
+* Let's Encrypt certificates location
+```bash
+export NUCYPHER_LETSENCRYPT_DIR=<DIRECTORY LOCATION>
```

Comment:
We should really rename this env var to `CERTIFICATE_DIR` or something similar, to keep it more general. @KPrasch wdyt?

If so we will also need to update the variable name in the `deploy/.env.template` file.

Reply:
I actually like the endorsement of Let's Encrypt here.